gate-web/config/gate.yml

```diff
@@ -81,13 +81,35 @@ spring:
       SORT_PROPERTIES_ALPHABETICALLY: true
     serialization:
       ORDER_MAP_ENTRIES_BY_KEYS: true
-#  oauth2:
-#      client:
-#        clientId:
-#        clientSecret:
-#        accessTokenUri:
-#        userAuthorizationUri:
-#        scope:
-#          - email
-#      resource:
-#        userInfoUri:
+
+---
+
+spring:
+  profiles: googleOAuth
+  oauth2:
+    client:
+      # Override these values in your own gate-googleOAuth.yml file
+      # clientId:
+      # clientSecret:
+      accessTokenUri: https://www.googleapis.com/oauth2/v4/token
+      userAuthorizationUri: https://accounts.google.com/o/oauth2/v2/auth?access_type=offline
+      scope: "profile email"
+    resource:
+      userInfoUri: https://www.googleapis.com/oauth2/v3/userinfo
+
+
+---
+
+spring:
+  profiles: azureOAuth
+  oauth2:
+    client:
+      # Override these values in your own gate-azureOAuth.yml file
+      # clientId:
+      # clientSecret:
+      # accessTokenUri: https://login.microsoftonline.com/<tenant_id>/oauth2/token
```

Comment:
It is when we setup the account that spinnaker runs under.  Scot and I need to document the right way to setup AD in order to use this.  That will take us a couple of days to flush out.  Stay tuned.

From: Travis Tomsu [mailto:notifications@github.com]
Sent: Thursday, May 12, 2016 12:38 PM
To: spinnaker/gate <gate@noreply.github.com>
Cc: Richard Guthrie <rguthrie@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [spinnaker/gate] Adds spring profile templates for Google and Azure OAuth providers. (#215)


In gate-web/config/gate.yml<https://github.com/spinnaker/gate/pull/215#discussion_r63083293>:

> +      userAuthorizationUri: https://accounts.google.com/o/oauth2/v2/auth?access_type=offline

> +      scope: "profile email"

> +    resource:

> +      userInfoUri: https://www.googleapis.com/oauth2/v3/userinfo

> +

> +

> +---

> +

> +spring:

> +  profiles: azureOAuth

> +  oauth2:

> +    client:

> +      # Override these values in your own gate-azureOAuth.yml file

> +      # clientId:

> +      # clientSecret:

> +      # accessTokenUri: https://login.microsoftonline.com/<tenant_id>/oauth2/token<https://login.microsoftonline.com/%3ctenant_id%3e/oauth2/token>

Done - is azureTenantId already a requirement to be set in spinnaker-local?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/spinnaker/gate/pull/215/files/4be5ca16eb86ec985495e28067dae3d854a253b7#r63083293>